ocaml/idl/markdown_backend.ml

```diff
@@ -161,9 +166,14 @@ let markdown_section_of_message printer ~is_class_deprecated ~is_class_removed x
     List.iter (fun p -> printer (get_param_row p)) x.msg_params;
     printer "";
 
-    printer (sprintf "_Return Type:_ `%s`" return_type);
+    printer ("_Return Type:_" ^ (if is_event_from then " an event batch" else sprintf " `%s`" return_type));
     printer "";
-    let descr= desc_of_ty_opt x.msg_result in
+    let descr =
+      if is_event_from then
+        "A structure consisting of a token ('token'), a map of valid references per object type ('valid_ref_counts'), and a set of event records ('events')."
```

Comment:
Long lines like this are difficult to review on GitHub.

Reply:
I agree, but it is using the style of the rest of the file, so I'd accept them in this case